src/Commands/SetShooter.cpp

```diff
@@ -0,0 +1,14 @@
+#include "SetShooter.h"
+#include "../Subsystems/Shooter.h"
+std::shared_ptr<Shooter> shooter = Shooter::getInstance();
```

Comment:
Is there a reason for this line? If you think this has a reason to be here, please tell me, because this is not needed.

This line should not be here. Please delete this.